### ISSUE TITLE
Spectating someone holding camera error

### ISSUE THREAD

**Issue by XxWestKillzXx** [Nayruden/Ulysses#393]
When you spectate someone that holds a camera it constantly spams the console with this error.

[ERROR] gamemodes/sandbox/entities/weapons/gmod_camera.lua:182: attempt to index local 'cmd' (a nil value)
  1. DoZoomThink - gamemodes/sandbox/entities/weapons/gmod_camera.lua:182
   2. unknown - gamemodes/sandbox/entities/weapons/gmod_camera.lua:171

**Comment by SticklyMan** [comment]
Duplicate of #328.

This was a bug with Garry's Mod, but has since been fixed on their dev branch. I'm not sure if they've pushed out the update quite yet, since they're waiting on other bugfixes.
